templates/helm/values.yaml.tpl

```diff
@@ -47,6 +47,10 @@ log:
 # If specified, the service controller will watch for object creation only in the provided namespace
 watchNamespace: ""
 
+# If specified, the service controller will use Role/Rolebinding instead of ClusterRole/ClusterRoleBindings
+# If set to true, user must set watchNamespace as well.
```

Comment:
You can actually force this by adding a `{{ required ... }}` block inside one of your conditionals.
See https://v2.helm.sh/docs/developing_charts/#using-the-required-function for information about required.

I could imagine you have, in :
```
{{ if not .Values.namespacedInstallation }}
...
{{ else }}
{{ required "watchNamespace must be set for namespaced installation" .Values.watchNamespace }}
...
```

Reply:
I'll into that and iterate